_specifications/lsp/3.17/language/inlineValue.md

```diff
@@ -178,10 +178,10 @@ export interface InlineValueVariableLookup {
  *
  * @since 3.17.0
  */
-export interface InlineValueEvaluatableExpression {
+export interface InlineValueEvaluableExpression {
```

Comment:
This is presumably an API break and wouldn't be accepted.

Reply:
Exactly, this can not be changed. It is named the same way in the VS Code API.